python/dglke/main.py

```diff
@@ -77,6 +78,19 @@ def create_encoder(args):
         else:
             raise NotImplementedError(f'init {args.init} is not implemented yet.')
         encoder = KGEEncoder(hidden_dim=args.hidden_dim,
+                             n_entity=args.n_entities,
+                             n_relation=args.n_relations,
+                             init_func=init_func,
+                             score_func=args.score_func)
+        return encoder
+    elif args.encoder == 'TransR':
+        from .nn.modules import TransREncoder
```

Comment:
You should import TransRencoder at the beginning if it does not introduce any strange dependencies.